### PR TITLE
correct outdir and name of multifreq files

### LINKED ISSUE #167: Multi-frequency saves files in the cwd instead of the outdir
## Expected Behavior

Files should be saved in the `outdir` directory given by the user.

## Actual Behavior

Files are saved outside of the `outdir` directory, i.e. the tests/data directory.

image

## Steps to Reproduce the Problem

```
    url = 'https://osf.io/9a7yv/download'
    test_path = resource_filename('phys2bids', 'tests/data')
    test_filename = 'Test_belt_pulse_multifreq.acq'
    test_full_path = os.path.join(test_path, test_filename)
    wget.download(url, test_full_path)
    test_chtrig = 3

    phys2bids(filename=test_filename, indir=test_path, outdir=test_path,
              chtrig=test_chtrig, num_timepoints_expected=1)
```

## Specifications

    - Python version: 3.7.6
    - phys2bids version: v1.3.0-beta+186.gfd98565.dirty
    - Platform: MacOS Catalina

## Possible solution

Pass the `outdir` argument to the part of the code that takes care of saving the multi-frequency ouput.

### LINKED ISSUE #168: Multi-frequency saves output files with incorrect (default) name
## Expected Behavior

Output files should be named based on the input filename.

## Actual Behavior

image

## Steps to Reproduce the Problem

See #167 .

## Specifications

    - Python version: 3.7.6
    - phys2bids version: v1.3.0-beta+186.gfd98565.dirty
    - Platform: MacOS Catalina

## Possible solution

We should pass the filename argument to the saving of these files.

### PR DESCRIPTION
**Pull request by vinferrer**
Closes #168 and  Closes #167

## Proposed Changes

  - Now multifrequency files have the name of the original files at the begging instead of outfile
  - Now multifrequency files go to the -outdir instead of $CWD

### CODE REVIEW

**Comment by vinferrer** [comment]
I now there may be a better way to link this to the heuristic file but for the moment this assures that the -outdir option is respected

**Comment by vinferrer** [comment]
so in the last push I just avoided that the file extension, for example `txt` appears in the output file

File: phys2bids/phys2bids.py [hunk 1]
```diff
@@ -268,7 +266,8 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
     # Create trigger plot. If possible, to have multiple outputs in the same
     # place, adds sub and ses label.
     LGR.info('Plot trigger')
-    plot_path = deepcopy(outfile)
+    plot_path = deepcopy(os.path.join(outdir,
```
Comment (smoia) [inline phys2bids/phys2bids.py]:
```suggestion
    plot_path = os.path.join(outdir,
```

File: phys2bids/phys2bids.py [hunk 2]
```diff
@@ -268,7 +266,8 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
...
+                         os.path.splitext(os.path.basename(filename))[0]))
```
Comment (smoia) [inline phys2bids/phys2bids.py]:
```suggestion
                         os.path.splitext(os.path.basename(filename))[0])
```

**Comment by vinferrer** [comment]
merging when test pass

**Pull request merged**